assets/src/edit-story/components/fontPicker/test/fontPicker.js

```diff
@@ -28,15 +28,15 @@ import APIContext from '../../../app/api/context';
 import { renderWithTheme } from '../../../testUtils';
 import fontsListResponse from './fontsResponse';
 
-async function getFontPicker() {
+async function getFontPicker({ onChange } = {}) {
   const getAllFontsPromise = Promise.resolve(fontsListResponse);
   const apiContextValue = {
     actions: {
       getAllFonts: () => getAllFontsPromise,
     },
   };
   const props = {
-    onChange: jest.fn(),
+    onChange: onChange || jest.fn(),
```

Comment:
Actually I'd prefer to just return the props along with the accessors [further down](https://github.com/google/web-stories-wp/pull/2313/files#diff-0ebe8ab039a00c9534c13e3b28b292f8R53), and then tests can ignore this `onChange` property or not as they please:

```js
    return { ...accessors, ...props };
```

Reply:
And thus remove the `onChange` argument from this function completely.

Reply:
On second thought I don't think we'd want to return props but rather make props an option that spreads since the get function also fetches the fonts.

```
const props = {
    onChange: onChange || jest.fn(),
    value: 'Roboto',
  };
```

I will spread any options into this.